DatabaseSetup.sql

```diff
@@ -0,0 +1,50 @@
+CREATE DATABASE JobForge_OdhranH;
+
+USE JobForge_OdhranH;
+
+
+CREATE TABLE Band(
+	BandName VARCHAR(20) PRIMARY KEY,
+    BandLevel VARCHAR(10),
+    Competencies VARCHAR(100)
+);
+
+CREATE TABLE Training(
+	TrainingName VARCHAR(20) PRIMARY KEY,
+    BandName VARCHAR(20),
+    TrainingType VARCHAR(20),
+    FOREIGN KEY(BandName) REFERENCES Band(BandName)
+);
+
+CREATE TABLE Employee(
+	EmployeeID INTEGER PRIMARY KEY,
+    EmployeeName VARCHAR(50),
+    EmployeePhoto VARCHAR(100),
+    EmployeeMessage VARCHAR(100)
+);
+
+CREATE TABLE Capability(
+	CapabilityName VARCHAR(30) PRIMARY KEY,
+    LeadID INTEGER,
+    FOREIGN KEY(LeadID) REFERENCES Employee(EmployeeID)
+);
+
+CREATE TABLE JobFamily(
+	JobFamily VARCHAR(30) PRIMARY KEY,
+    CapabilityName VARCHAR(30),
+    FOREIGN KEY(CapabilityName) REFERENCES Capability(CapabilityName)
+);
+
+CREATE TABLE JobRole(
+	RoleName VARCHAR(20) PRIMARY KEY,
+    Specification VARCHAR(100),
+    CapabilityName VARCHAR(20),
+    BandName VARCHAR(20),
+    Responsibilities VARCHAR(100),
+    SharepointLink VARCHAR(100),
+    FOREIGN KEY(BandName) REFERENCES Band(BandName),
+    FOREIGN KEY(CapabilityName) REFERENCES Capability(CapabilityName)
+);
+
+
+GRANT ALL privileges on `JobForge_OdhranH`.* to JosephM, CriofanMc, ConorT;
```

Comment:
I don't think this can be included here. Feels like a security issue. You are exposing a list of database users that exist in the database. Also you don't have permission to run that line. So there are two requirements this breaks...
1. No usernames/passwords in source
2. Must be able to recreate database in the case of an outage
https://tasks.office.com/kainos.com/Home/Task/jVDJm11QNUS0vk5XGA1xh5YAIbSt?Type=TaskLink&Channel=Link&CreatedTime=638373763633270000